### ISSUE TITLE
[Bug]: selefra apply bug

### ISSUE THREAD

**Issue by srikwang** [selefra/selefra#10]
### Contact Details

cheng.wang@cloudfall.cn

### Version

0.2.1

### What happened?

I try to test selefra in centos8, I create selefra users, initialize services, selefra test is all right, but I encounter the following problems when selefra apply

### Relevant log output

```shell
bash-4.4$ selefra apply 
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
github.com/selefra/selefra/pkg/modules/parser.(*YamlFileToModuleParser).Parse(0xc0000e62b8)
       /home/runner/work/selefra/selefra/pkg/modules/parser/yaml_file_to_module_parser.go:58 +0x9ae
github.com/selefra/selefra/pkg/modules/module_loader.(*LocalDirectoryModuleLoader).Load(0xc0000121d0, {0x17e5558, 0xc000046088})
       /home/runner/work/selefra/selefra/pkg/modules/module_loader/local_directory_module_loader.go:103 +0x565
github.com/selefra/selefra/pkg/modules/executors.(*ProjectLocalLifeCycleExecutor).loadModule(0xc0000e61c8, {0x17e5558, 0xc000046088})
       /home/runner/work/selefra/selefra/pkg/modules/executors/project_local_life_cycle_executor.go:270 +0x34c
github.com/selefra/selefra/pkg/modules/executors.(*ProjectLocalLifeCycleExecutor).Execute(0xc0000e61c8, {0x17e5558, 0xc000046088})
       /home/runner/work/selefra/selefra/pkg/modules/executors/project_local_life_cycle_executor.go:128 +0xb1
github.com/selefra/selefra/cmd/apply.Apply({0x17e5558, 0xc000046088}, 0xc0000eea80, {0x14324c0, 0x2}, {0xc0000bc960, 0x20})
       /home/runner/work/selefra/selefra/cmd/apply/apply.go:66 +0x20b
github.com/selefra/selefra/cmd/apply.NewApplyCmd.func1(0xc0002aac00, {0x1433551?, 0x0?, 0x0?})
       /home/runner/work/selefra/selefra/cmd/apply/apply.go:31 +0xe5
github.com/spf13/cobra.(*Command).execute(0xc0002aac00, {0x21b27f8, 0x0, 0x0})
       /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:916 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0x216ca20)
       /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
       /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
github.com/selefra/selefra/cmd.Execute()
       /home/runner/work/selefra/selefra/cmd/root.go:76 +0x46
main.main()
       /home/runner/work/selefra/selefra/main.go:23 +0x17
```

**Comment by FelixsJiang** [comment]
We are already aware of this problem, and the dev-gpt branch has been fixed, and the v0.2.2 official version will release a binary version that fixes this problem.

**Comment by FelixsJiang** [comment]
We fixed this issue in this pr https://github.com/selefra/selefra/pull/11/files
